lua/grpc-hook.lua

```diff
@@ -33,6 +33,10 @@ local function init()
     _G.GRPC = nil
   end
 
+  function handler.onPlayerTrySendChat(playerID, msg, all)
```

Comment:
The docs about `onPlayerTrySendChat` say:

> they are called directly from the network code, so try to make them as fast as possible

And this is unfortunately true. The handler blocks the server until `onPlayerTrySendChat` returns. This is why we might not wanna keep the POC.